Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -78,6 +81,7 @@ import BraintreeDataCollector
         completion: @escaping (BTPayPalAccountNonce?, Error?) -> Void
     ) {
         tokenize(request: request, completion: completion)
+        isVaultRequest = true
```

Comment:
We need to set `isVaultRequest` before calling the `tokenize` method.

```suggestion
        isVaultRequest = true
        tokenize(request: request, completion: completion)
```